docs/source/developers/deployment.rst

```diff
@@ -5,13 +5,13 @@ Deployment
 Container Repository
 --------------------
 
-In order to use the container image that you just built in the previous step, the image needs to be pushed to some container repository.
+In order to more easily consume the container image that you just built, the images needs to be pushed to some container repository.
```

Comment:
The `container image` sounds awkward, but I think it's valid. Just noting  a potential confusion point.

Reply:
I will try and reword